src/act/node/canister_method/query_or_update_definition.rs

```diff
@@ -41,15 +41,15 @@ impl QueryOrUpdateDefinition {
         }
     }
 
-    pub fn generate_function_body(&self, keyword_list: &Vec<String>) -> TokenStream {
+    pub fn generate_function_body(&self, context: &Context) -> TokenStream {
```

Comment:
Now that we have the full context here, what if we removed cdk_name from the QueryOrUpdateDefinition struct?